src/firefly/js/tables/reducer/TableDataReducer.js

```diff
@@ -20,21 +20,22 @@ export function dataReducer(state={data:{}}, action={}) {
                 return updateSet(root, [tbl_id, 'selectInfo'], selectInfo);
             } else return root;
 
-        case (Cntlr.TABLE_NEW_LOADED)  :
+        case (Cntlr.TABLE_LOADED)  :
             const statusPath = [tbl_id, 'tableMeta', 'Loading-Status'];
             if (get(root, statusPath, 'COMPLETED') !== 'COMPLETED') {
                 return updateSet(root, statusPath, 'COMPLETED');
             } else return root;
 
         case (Cntlr.TABLE_HIGHLIGHT)  :
         case (Cntlr.TABLE_UPDATE)  :
-            return TblUtil.smartMerge(root, {[tbl_id] : action.payload});
+            return TblUtil.smartMerge(root, {[tbl_id] : {isFetching:false, ...action.payload}});
```

Comment:
Just curious, why isFetching set to false here? I though on TABLE_UPDATE the data are still coming. Or isFetching marks the state before first TABLE_UPDATE?